lib/airbrake/rack/user.rb

```diff
@@ -50,7 +50,12 @@ def as_json
       private
 
       def try_to_get(key)
-        String(@user.__send__(key)) if @user.respond_to?(key)
+        String(@user.__send__(key)) if can_call_method?(key)
```

Comment:
@valdis Sorry to read this. My comment was well-intentioned and, it's not a matter of aesthetic taste. Anything that goes into a library of this kind gets called over and over and eventually does affect performance. Peer review is good.